def fail_sometimes():
    # div_by_zero = 1/0
    return 'No Exception!'
